Mathlib/CategoryTheory/EpiMono.lean

```diff
@@ -168,6 +168,46 @@ theorem IsIso.of_epi_section {X Y : C} (f : X ⟶ Y) [hf : IsSplitEpi f] [hf' :
     IsIso f :=
   @IsIso.of_epi_section' _ _ _ _ _ hf.exists_splitEpi.some hf'
 
+def SplitMono.op {X Y : C} {f : X ⟶ Y} (sm : SplitMono f) : SplitEpi f.op where
+  section_ := sm.retraction.op
+  id := by
+    obtain ⟨ret, _⟩ := sm
+    observe: ret.op ≫ f.op = 𝟙 _
+    exact this
+
+def SplitMono.unop {X Y : C} {f : Opposite.op Y ⟶  Opposite.op X} (sm : SplitMono f) :
```

Comment:
It would be better if `f : Y ⟶ X` was a morphisms between objects `Y` and `X` of the opposite category of `C`, instead of `op Y ⟶ op X` with `X` and `Y` in `C`.